lib/events/s3sessions/s3stream.go

```diff
@@ -48,6 +48,9 @@ func (h *Handler) CreateUpload(ctx context.Context, sessionID session.ID) (*even
 	if !h.Config.DisableServerSideEncryption {
 		input.ServerSideEncryption = aws.String(s3.ServerSideEncryptionAwsKms)
 	}
+	if h.Config.ACL != "" {
+		input.ACL = aws.String(h.Config.ACL)
+	}
```

Comment:
Can you check if this is needed? I think you're setting it on the AWS session, so it shouldn't be needed right?

Reply:
Looks like it might be needed, but please verify.

https://docs.aws.amazon.com/AmazonS3/latest/API/API_CreateMultipartUpload.html

Reply:
This is needed. The session doesn't handle the ACL.